.github/workflows/ci_build_test.yml

```diff
@@ -2,13 +2,17 @@ name: CI Build
 
 on: [push, pull_request]
 
+env:
+  RUST_TEST_TIME_UNIT: 150,5000
+  RUST_TEST_TIME_INTEGRATION: 150,5000
+
 jobs:
   build:
     name: Build and Test
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest, macos-latest]
+        os: [ubuntu-20.04, windows-latest, macos-latest]
```

Comment:
Should also change the `ubuntu-latest` usage in https://github.com/partiql/partiql-lang-rust/blob/6c3bb697bde5152efeff60bd32f595879f26e70b/.github/workflows/coverage.yml#L18 to `ubuntu-20.04` to make the "Coverage" step succeed